sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/UserFreeUpRequestValidator.java

```diff
@@ -0,0 +1,87 @@
+package org.sunbird.common.request;
+
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.ProjectUtil;
+import org.sunbird.common.responsecode.ResponseCode;
+
+import java.text.MessageFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class UserFreeUpRequestValidator extends BaseRequestValidator {
+
+    private Request request;
+    private static List<String> identifiers = new ArrayList<>();
+    static {
+        identifiers.add("email");
```

Comment:
use already define constant for email and phone

Reply:
done